src/accordion.test.visuals.ts

```diff
@@ -1,78 +1,18 @@
 import { expect, test } from '@playwright/test';
 import type GlideCoreAccordion from './accordion.js';
 
-test.describe('accordion--accordion', () => {
-  test.describe('globals=theme:light', () => {
-    test('open=${false}', async ({ page }, test) => {
-      await page.goto(`?id=${test.titlePath.at(1)}&${test.titlePath.at(2)}`);
-      await page.locator('glide-core-accordion').waitFor();
-      await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
-    });
-
-    test('open=${true}', async ({ page }, test) => {
-      await page.goto(`?id=${test.titlePath.at(1)}&${test.titlePath.at(2)}`);
-
-      await page
-        .locator('glide-core-accordion')
-        .evaluate<void, GlideCoreAccordion>((element) => {
-          element.open = true;
-        });
-
-      await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
-    });
-  });
-
-  test.describe('globals=theme:dark', () => {
-    test('open=${false}', async ({ page }, test) => {
-      await page.goto(`?id=${test.titlePath.at(1)}&${test.titlePath.at(2)}`);
-      await page.locator('glide-core-accordion').waitFor();
-      await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
-    });
-
-    test('open=${true}', async ({ page }, test) => {
-      await page.goto(`?id=${test.titlePath.at(1)}&${test.titlePath.at(2)}`);
-
-      await page
-        .locator('glide-core-accordion')
-        .evaluate<void, GlideCoreAccordion>((element) => {
-          element.open = true;
-        });
-
-      await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
-    });
-  });
-});
-
-test.describe('accordion--with-icons', () => {
-  test.describe('globals=theme:light', () => {
-    test('open=${false}', async ({ page }, test) => {
-      await page.goto(`?id=${test.titlePath.at(1)}&${test.titlePath.at(2)}`);
-      await page.locator('glide-core-accordion').waitFor();
-      await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
-    });
-
-    test('open=${true}', async ({ page }, test) => {
-      await page.goto(`?id=${test.titlePath.at(1)}&${test.titlePath.at(2)}`);
-
-      await page
-        .locator('glide-core-accordion')
-        .evaluate<void, GlideCoreAccordion>((element) => {
-          element.open = true;
-        });
-
-      await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
-    });
-  });
+const stories = JSON.parse(process.env.STORIES ?? '');
 
-  test.describe('globals=theme:dark', () => {
+for (const story of stories.Accordion) {
```

Comment:
Now iterate through every story.

Reply:
`story === '?id=split-button--split-button&globals=theme:light'`

`story` is slightly odd in that it's not just a story but a theme as well 🤔.

Reply:
Struggling to think of a more suitable name 😅.  I think `story` is fine unless you think of something different. I think using `stories.Accordion` and `story` is pretty clear that we are testing each Accordion story. Obviously mixing in the theme complicates things, but I'm not sure we'll have that many additional configuration items similar to theme in the short-term.